packages/template-retail-react-app/app/components/icons/index.jsx

```diff
@@ -183,7 +183,7 @@ export const PlugIcon = icon('plug')
 export const PlusIcon = icon('plus')
 export const MastercardIcon = icon('cc-mastercard', {viewBox: MastercardSymbol.viewBox})
 export const ReceiptIcon = icon('receipt')
-export const SearchIcon = icon('search')
+export const SearchIcon = icon('search', {'aria-hidden': true})
```

Comment:
One of the ACs is to make this configurable. Image there is a case that we want icon to be visible by screen reader, how would we do that?

Reply:
> This is in the template retail react app so if a project wanted to change this, they could modify the line that is highlighted here.
Do you mean override this file here?

Reply:
> the ticket itself describes this icon as "purely decorative and that it does not offer any additional value"
It is decorative for the search input currently, but the customer may use this icon elsewhere with different purposes, so we can't be sure about that

Reply:
Yes, due to how overrides currently are, overriding this property would require adding an overrides file into the project extension and redefining the SearchIcon export there.

Reply:
As for using the icon in a place where it can be focused, a project can create a separate export like so:

`export const SearchIconFocusable = icon('search', {'aria-hidden': false, focusable: true})`